### PR TITLE
Practice git

### PR DESCRIPTION
**Pull request by prabhat1081**

### CODE REVIEW

**Comment by krishna95** [comment]
@prabhat1081 Resolve Merge Conflicts, a better commit message.

**Comment by abinashmeher999** [comment]
This thing shouldn't take more than one commit. And also the name appears on the same line.

**Comment by abinashmeher999** [comment]
You do not open a new PR to correct the mistakes. I am reopening the PR.

**Comment by narayanaditya95** [comment]
@prabhat1081 Make your branch even with AGV-IIT-KGP/git-sandbox:master. Resolve those merge conflicts. Rebase commits into one with better unique commit meassage.

**Comment by hargup** [comment]
Hi @prabhat1081  we have changed the format to enter your name, see the commit message of https://github.com/AGV-IIT-KGP/git-sandbox/commit/095b8a39ab812ffa5b1fe933208dcbddcfff7f17 to know the new format and the reasons for the changes.

You need to rebase your pull request with the latest master. To know more about git-rebasing read http://git-scm.com/book/en/v2/Git-Branching-Rebasing Please don't hesitate to ask questions when you can't figure things out. And as @narayanaditya95 mentione, you should have only one commit for this change.

**Comment by routeaccess** [comment]
@prabhat1081 Please notice that the format of entering your details has changed again (as per pull request #26) , causing your changes to be stale/obsolete. Please rebase your changes to the latest commit in master branch. 
[Here](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) is a detailed explanation.
In addition:
- Please squash your commits into a single commit, as explained in the given link. (You currently have 5 commits, not 2 commits waiting to be merged). Also note that it is not a good practice to have merge commits within a PR.
- Edit the PR description above to something more descriptive.
Refer to previous commits and pull-requests for reference.

**Comment by krishna95** [comment]
@prabhat1081 Please squash the commits into one using rebase -i master

**Comment by abinashmeher999** [comment]
@krishna95 I think it's ok even if there are 2 commits.

**Comment by hargup** [comment]
@prabhat1081 Yes you need to squash the commits.

@abinashmeher999 such things can be acceptable when working on code but this is a exercise to learn git and @prabhat1081 needs to clean up the history.

File: README.md [hunk 1]
```diff
@@ -22,3 +22,5 @@ and the lines starting with `*` are automatically rendered as bullet points.
 * Harsh Gupta [hargup](http://github.com/hargup)
 * Krishna Bagadia [krishna95](http://github.com/krishna95)
 * Nevin Valsaraj [routeaccess](http://github.com/routeaccess)
+* Prabhat Agarwal [prabhat1081](http://github.com/prabhat1081)
+
```
Comment (hargup) [inline README.md]: Please remove this newline.

**Comment by routeaccess** [comment]
Looks good to go. 
@hargup Please merge.

**Comment by hargup** [comment]
:tada:

**Pull request merged**